tests/tests/test_mtls.py

```diff
@@ -187,6 +192,8 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
         tmpdir = tempfile.mkdtemp()
 
         ssl_engine_id = "pkcs11"
+        if not use_engine:
+            ssl_engine_id = ""
```

Comment:
Doesn't the following read simpler?

```suggestion
        ssl_engine_id = ""
        if use_engine:
            ssl_engine_id = "pkcs11"
```

Reply:
It does indeed